frontend/src/BudgetCard.tsx

```diff
@@ -0,0 +1,225 @@
+        <Card sx={{ mb: 2, maxWidth: "600px" }} variant="plain">
+            <Grid
+                container
+                columnSpacing={1}
+                rowSpacing={2}
+                columns={numColumns - hiddenColumns.size}
+                alignItems="center"
+                sx={{ flexGrow: 1 }}
+            >
+                {
+                    <>
+                        <BudgetRow hiddenColumns={hiddenColumns}>
+                            <div />
+                            <HeaderLabel text="Budget" />
+                            <HeaderLabel text="Reserved" />
+                            <HeaderLabel text="Remaining" />
+                        </BudgetRow>
+
+                        {budgetFilters.has(BudgetFilter.RESOURCES) && (
+                            <BudgetRow hiddenColumns={hiddenColumns}>
+                                <HeaderLabel text="Resources" />
+                                <BudgetInput
+                                    label="Resource Budget"
+                                    isLabelVisible={false}
+                                    value={resourceBudget}
+                                    onChange={setResourceBudget}
+                                />
+                                <div />
+                                <OverUnderNumber
+                                    value={
+                                        resourceBudget -
+                                        getTotalCost(unitCounts)
+                                    }
+                                />
+                            </BudgetRow>
+                        )}
+
+                        {budgetFilters.has(BudgetFilter.PRODUCTION_LIMIT) && (
+                            <BudgetRow hiddenColumns={hiddenColumns}>
+                                <HeaderLabel text="Production Limit" />
+                                <BudgetInput
+                                    label="Production Limit Budget"
+                                    isLabelVisible={false}
+                                    value={capacityBudget}
+                                    onChange={setCapacityBudget}
+                                />
+                                <div />
+                                <OverUnderNumber
+                                    value={
+                                        capacityBudget -
+                                        sumUnitCounts(unitCounts)
+                                    }
+                                />
+                            </BudgetRow>
+                        )}
+
+                        {budgetFilters.has(BudgetFilter.FLEET_SUPPLY) && (
+                            <BudgetRow hiddenColumns={hiddenColumns}>
+                                <HeaderLabel text="Fleet Supply" />
+                                <BudgetInput
+                                    label="Fleet Supply Budget"
+                                    isLabelVisible={false}
```

Comment:
All the input labels are invisible now since the grid format makes them visually unnecessary, but still passing a label that gets converted to an `aria-label` for accessibility

Reply:
Oooh yes good call, updated!